config/rollup.config.aliases.js

```diff
@@ -0,0 +1,12 @@
+import memory from 'rollup-plugin-memory';
+import rollupConfig from './rollup.config';
+
+export default Object.assign({}, rollupConfig, {
+	plugins: [
+		memory({
+			path: 'src/preact',
+			contents: `import { h } from './preact';export * from './preact';export { h as createElement };`
```

Comment:
Wondered about that too but I preferred sticking to the default rollup config in order to minimize the complexity added by this PR. Moreover I think there shouldn't be anything more from basic React's API that isn't in `preact`: we only hit an issue with `createElement` because it's React's implementation of jsx's `h`, which in the end isn't standard.

However still this code as a literal is pretty ugly :mask: and I would be ok with isolating it. You've got the last word: should I do it? :athletic_shoe: